p8e-api/src/main/kotlin/io/provenance/engine/index/ProtoIndexer.kt

```diff
@@ -41,21 +44,25 @@ class ProtoIndexer(
                             .first { keyPairs.containsKey(it.encryptionPublicKey.toHex()) }
                             .let { keyPairs.getValue(it.encryptionPublicKey.toHex()) }
 
+                        // Need a reference of the signer that is used to verify signatures.
+                        val signer = transaction { affiliateService.getSigner(keyPair.public) }
```

Comment:
looks like you were able to get rid of the transaction around the getSigner call on line 78 of ObjectGrpc (looks like there isn't any db stuff in that function ?), can you nix it everywhere else as well?

Reply:
ill take care of it

Reply:
actually just saw line 78 of object grpc, its wrapped with a transaction one line above

Reply:
oh yeah, missed that... also just looked at the getSigner function, and I guess it does fetch the affiliate to determine which signer to use, so :nothingtodohere: